android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">GoodDollar</string>
+    <string moduleConfig="true" name="CodePushDeploymentKey">u82UoKv-RLJwp6_D1v3o-uGg_imkSAbuJ7v2S</string>
```

Comment:
could it be configured via some settings file (we could generate it during github actions runs) ?

Reply:
yeah, we need to discuss it. @johnsmith-gooddollar . I am finding it.